deployment/terraform/variables.tf

```diff
@@ -214,6 +214,42 @@ variable "rds_cpu_credit_balance_threshold" {
   type    = number
 }
 
+variable "batch_root_block_device_size" {
+  default = 32
+  type    = number
+}
+
+variable "batch_root_block_device_type" {
+  default = "gp2"
```

Comment:
Were you able to test `gp3`? Not necessarily for the benchmark, but just to see if making use of it is as easy as swapping it in here.

Reply:
I left it out for the benchmarking since it was a new variable to contend with. However, after testing, it is as easy as swapping it in so I added it.